calculator/main.py

```diff
@@ -26,7 +31,22 @@ def perform_operation(calc, operation, params):
     :param params: Tuple containing the list of nums to operate with.
                    ie: (1, 2, 3, 4.5, -2)
     """
-    pass
+    if operation not in calc['operations'].keys():
```

Comment:
I don't think you need the `.keys()` here. --> `if operation not in calc['operations']`